quality_of_service_demo/README.md

```diff
@@ -0,0 +1,57 @@
+# Quality of Service Demos
+
+The demo applications in this package show the various Quality of Service settings available in ROS2. The intent is to provide a dedicated application for each QoS type, in order to highlight that single feature in isolation.
+
+## History
+No History demo app exists yet in this package
+
+## Reliability
+No Reliability demo app exists yet in this package
+
+## Durability
+No Durability demo app exists yet in this package
+
+## Deadline
+`quality_of_service_demo/deadline` demonstrates messages not meeting their required deadline.
+
+The application requires an argument a `deadline_duration` - this is the maximum acceptable time (in milliseconds) between messages published on the topic.
```

Comment:
Is this an integer or a float, what happens if it is negative or zero? Maybe just mention that `this is the maximum acceptable time (in positive, non-zero, integer milliseconds) ...`.